Demo/Components/SelfDeclaration/SelfDeclarationDemoView.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  Copyright © 2019 FINN AS. All rights reserved.
+//
+
+import FinniversKit
+
+class SelfDeclarationDemoView: UIView {
+
+    private lazy var selfDeclarationView = SelfDeclarationView(withAutoLayout: true)
+    private lazy var scrollView = UIScrollView(withAutoLayout: true)
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+    }
+
+    public required init?(coder aDecoder: NSCoder) { fatalError() }
+
+    private func setup() {
+        addSubview(scrollView)
+        scrollView.fillInSuperview()
+        scrollView.addSubview(selfDeclarationView)
+
+        NSLayoutConstraint.activate([
+            selfDeclarationView.leadingAnchor.constraint(equalTo: leadingAnchor, constant: .mediumLargeSpacing),
+            selfDeclarationView.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -.mediumLargeSpacing),
+            selfDeclarationView.topAnchor.constraint(equalTo: scrollView.topAnchor, constant: .mediumLargeSpacing),
+            selfDeclarationView.bottomAnchor.constraint(equalTo: scrollView.bottomAnchor, constant: -.mediumLargeSpacing),
+        ])
+
```

Comment:
i guess you could do something like

```swift
scrollView.layoutMargins = UIEdgeInsets(all: .mediumLargeSpacing)
selfDeclarationView.fillInSuperviewMargins()
```

the demo view's leading and trailing anchors will be the same as the scroll view

Reply:
It works to constrain the view, but then it's not possible to scroll in the scrollView :/